app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/importlogins/handlers/ImportLoginHandler.kt

```diff
@@ -50,7 +51,10 @@ data class ImportLoginHandler(
             },
             onRetrySync = { viewModel.trySendAction(ImportLoginsAction.RetryVaultSync) },
             onFailedSyncAcknowledged = {
-                viewModel.trySendAction(ImportLoginsAction.FailSyncAcknowledged)
+                viewModel.trySendAction(ImportLoginsAction.FailedSyncAcknowledged)
+            },
+            onSuccessfulSyncAcknowledged = {
+                viewModel.trySendAction(ImportLoginsAction.SuccessfulSyncAcknowledged)
```

Comment:
Can we add a test that asserts when this action is fired

Reply:
https://github.com/bitwarden/android/pull/4125#discussion_r1809523524